client/src/main/java/com/battleships/gamewindow/board/OpponentBoard.java

```diff
@@ -0,0 +1,7 @@
+package com.battleships.gamewindow.board;
+
+public class OpponentBoard extends Board {
+    public OpponentBoard() {
```

Comment:
What is the difference between the Board and Opponent board?

Reply:
Need time to implement it :)